### DISCUSSION TITLE
Bluetooth for both versions | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1006] [Q&A]
# Bluetooth for both versions | Seeed Studio Wiki

Bluetooth usage with Seeed Studio XIAO ESP32S3.

https://wiki.seeedstudio.com/xiao_esp32s3_bluetooth/

**Comment by Tamirsres** [comment]
Hi,
I have 2 modules, how can I use BLE with light sleep?
